From f0c8841b86cce94d70257b096bab924879173820 Mon Sep 17 00:00:00 2001
From: Jarno Rajahalme <jarno@covalent.io>
Date: Thu, 28 Jan 2021 12:49:47 -0800
Subject: [PATCH] bazel: Assume default compiler is gcc

It is impossible to turn on the "//bazel/gcc_build" config setting, see:

   https://github.com/bazelbuild/bazel/issues/12707

GCC 9, which is the default in Ubuntu 20.04, requires the -Wno-maybe-uninitialized for
@com_googlesource_quiche//:quic_core_framer_lib to compile successfully.

Mitigate by assuming the default "compiler" is GCC.

Signed-off-by: Jarno Rajahalme <jarno@covalent.io>
---
 bazel/BUILD | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bazel/BUILD b/bazel/BUILD
index e3829e17f8..fc164997cc 100644
--- a/bazel/BUILD
+++ b/bazel/BUILD
@@ -169,7 +169,7 @@ config_setting(
 config_setting(
     name = "gcc_build",
     flag_values = {
-        "@bazel_tools//tools/cpp:compiler": "gcc",
+        "@bazel_tools//tools/cpp:compiler": "compiler",
     },
 )
 
-- 
2.24.3 (Apple Git-128)

